skimage/transform/_geometric.py

```diff
@@ -1208,7 +1214,7 @@ class EuclideanTransform(ProjectiveTransform):
         higher dimensions, you must provide or estimate the transformation
         matrix.
     translation : sequence of float, length D, optional
-        Translation parameters for each axis.
+        x, y[, z, ...] translation parameters.
```

Comment:
Can't suggest here, but how about
```
    translation : (x, y[, z, ...]) sequence of float, length D, optional
        Translation parameters for each axis.
```

Reply:
I made that change. I had been copying the doc for SimilarityTransform, but it looks like both EuclideanTransform and AffineTransformation follow your suggestion. We could change SimilarityTransform to be consistent.